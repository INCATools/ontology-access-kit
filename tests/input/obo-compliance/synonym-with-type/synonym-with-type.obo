format-version: 1.4
ontology: synonym-with-type

[Term]
id: X:1
name: example term
synonym: "synonym with type" RELATED ST:1 []

