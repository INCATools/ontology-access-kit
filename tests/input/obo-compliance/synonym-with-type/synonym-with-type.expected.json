{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/synonym-with-type.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "lbl" : "example term",
      "type" : "CLASS",
      "meta" : {
        "synonyms" : [ {
          "synonymType" : "http://purl.obolibrary.org/obo/ST_1",
          "pred" : "hasRelatedSynonym",
          "val" : "synonym with type"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym",
      "lbl" : "has_related_synonym",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasSynonymType",
      "lbl" : "has_synonym_type",
      "type" : "PROPERTY"
    } ]
  } ]
}